forge-gui/res/cardsfolder/upcoming/springheart_nantuko.txt

```diff
@@ -0,0 +1,13 @@
+Name:Springheart Nantuko
+ManaCost:1 G
+Types:Enchantment Creature Insect Monk
+PT:1/1
+K:Bestow:1 G
+S:Mode$ Continuous | Affected$ Card.EnchantedBy | AddPower$ 1 | AddToughness$ 1 | Description$ Enchanted creature gets +1/+1.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Land.YouCtrl | TriggerZones$ Battlefield | Execute$ TrigBranch | TriggerDescription$ Landfall — Whenever a land enters the battlefield under your control, you may pay {1}{G} if CARDNAME is attached to a creature you control. If you do, create a token that's a copy of that creature. If you didn't create a token this way, create a 1/1 green Insect creature token.
+SVar:TrigBranch:DB$ Branch | BranchConditionSVar$ X | BranchConditionSVarCompare$ GE1 | TrueSubAbility$ DBCopyNotPaid | FalseSubAbility$ DBToken
+SVar:DBCopyNotPaid:DB$ Token | UnlessCost$ 1 G | UnlessPayer$ You | UnlessResolveSubs$ WhenPaid | UnlessAI$ Never | SubAbility$ DBCopy | TokenAmount$ 1 | TokenScript$ g_1_1_insect | TokenOwner$ You

```

Comment:
While technically correct I think you could probably clean up the logic which would also save 2 effect lines if instead of Branch check you combine the cost payment with ConditionPresent$ and UnlessSwitched$ (but on CopyPermanent then)

Reply:
done